### DISCUSSION TITLE
Problem with searchable belongs_to

### DISCUSSION THREAD

**Discussion by IgorPrzybylski12** [avo-hq/avo#3498] [Help]
Why after adding "searchable: true:" nothing changes, there is still the same select that was there before the change 
    field :gene, as: :belongs_to, searchable: true, sortable: -> {
      query.order(id: direction)
    }

**Comment by Paul-Bob** [comment]
Hi @IgorPrzybylski12. 

The `searchable` feature is included with the Pro license.

If you don't have a Pro or Advanced license, the `searchable` option won't have any effect.

However, if you already have a Pro or Advanced license and the `searchable` option still isn't working, let us know!
